### PR TITLE
Improve install process

### PR DESCRIPTION
**Pull request by Pytal**
Now with the satisfaction of clicking a button :)

| Before | After |
| ---       | ---     |
|  |  |
|  |  |

### CODE REVIEW

File: core/src/components/setup/RecommendedApps.vue [hunk 1]
```diff
@@ -110,27 +118,30 @@ export default {
...
+				})
+				.catch(error => {
+					logger.error('could not fetch app list', { error })
+
+					this.loadingAppsError = true
+				})
+				.then(() => {
+					this.loadingApps = false
+				})
+		},
```
Comment (artonge) [inline core/src/components/setup/RecommendedApps.vue:144]: Can we use async/await and try/catch here ?

**Review by artonge — CHANGES_REQUESTED** [review]

**Comment by artonge** [comment]
I would say having the list of recommended apps before starting the install is nice too. Not sure if we should go on with this change.

**Review by PVince81 — APPROVED** [review]
:+1: code looks fine

optionally see Louis' comment

File: core/src/components/setup/RecommendedApps.vue (hunk 1 above)
Comment (Pytal) [reply core/src/components/setup/RecommendedApps.vue]: `async`hronized

Comment (Pytal) [reply core/src/components/setup/RecommendedApps.vue]: `async`hronized :)

**Comment by Pytal** [comment]
Now updated to show the recommended apps alongside the install button

**Review by artonge — APPROVED** [review]
Code looks code !

**Review by szaimen — APPROVED** [review]
Seems to work 👍

Maybe we could think about removing the checkbox now that you need to press a button to start the installation of the apps either way:
image

Also we could think about renaming the desciption of the button from `Finish installation` to `Continue installation` as this is not the last step if you don't uncheck the `Install recommended apps` checkbox?

**Comment by Pytal** [comment]
Checkbox removed and wording updated, plus updated screenshots

**Review by szaimen — APPROVED** [review]
Works 👍 
(but didn't review the code)

Thanks for adressing my comment :)

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**